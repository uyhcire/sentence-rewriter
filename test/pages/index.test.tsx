import React from 'react'
import { render } from '../testUtils'
import { Home } from '../../pages/index'

describe('Home page', () => {
  beforeEach(() => {
    window.getSelection = jest.fn()
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
