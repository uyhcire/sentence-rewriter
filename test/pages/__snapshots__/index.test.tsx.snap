// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="jsx-2250570153 container"
  >
    <main
      class="jsx-2250570153"
    >
      <h1
        class="jsx-2250570153 title"
        style="padding-bottom: 40px;"
      >
        In Other Words üñáÔ∏è
      </h1>
      <p
        class="jsx-2250570153"
      >
        Press 
        <strong
          class="jsx-2250570153"
        >
          <kbd
            class="jsx-2250570153"
          >
            Command + Quote
          </kbd>
        </strong>
         to select a sentence
        <br
          class="jsx-2250570153"
        />
        Press 
        <strong
          class="jsx-2250570153"
        >
          <kbd
            class="jsx-2250570153"
          >
            Command + Enter
          </kbd>
        </strong>
         to rephrase it
      </p>
      <div
        class="jsx-2250570153"
        style="width: 800px; height: 600px; padding-top: 50px; padding-bottom: 50px;"
      >
        <div
          style="border: solid; border-color: lightblue; width: 100%; height: 100%; padding: 10px;"
        >
          <div
            contenteditable="true"
            data-gramm="false"
            data-slate-editor="true"
            data-slate-node="value"
            role="textbox"
            style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
          >
            <div
              data-slate-node="element"
              style="position: relative;"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    This is good. But that is bad. And it's important that you know the difference.
                  </span>
                </span>
              </span>
            </div>
            <div
              data-slate-node="element"
              style="position: relative;"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-length="0"
                    data-slate-zero-width="n"
                  >
                    Ôªø
                    <br />
                  </span>
                </span>
              </span>
            </div>
            <div
              data-slate-node="element"
              style="position: relative;"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    Another day, another paragraph. I can't wait for tomorrow!
                  </span>
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="jsx-2250570153 grid"
      >
        <a
          class="jsx-2250570153 card"
          href="https://nextjs.org/docs"
        >
          <h3
            class="jsx-2250570153"
          >
            Documentation ‚Üí
          </h3>
          <p
            class="jsx-2250570153"
          >
            Find in-depth information about Next.js features and API.
          </p>
        </a>
        <a
          class="jsx-2250570153 card"
          href="https://nextjs.org/learn"
        >
          <h3
            class="jsx-2250570153"
          >
            Learn ‚Üí
          </h3>
          <p
            class="jsx-2250570153"
          >
            Learn about Next.js in an interactive course with quizzes!
          </p>
        </a>
        <a
          class="jsx-2250570153 card"
          href="https://github.com/vercel/next.js/tree/master/examples"
        >
          <h3
            class="jsx-2250570153"
          >
            Examples ‚Üí
          </h3>
          <p
            class="jsx-2250570153"
          >
            Discover and deploy boilerplate example Next.js projects.
          </p>
        </a>
        <a
          class="jsx-2250570153 card"
          href="https://vercel.com/import?filter=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        >
          <h3
            class="jsx-2250570153"
          >
            Deploy ‚Üí
          </h3>
          <p
            class="jsx-2250570153"
          >
            Instantly deploy your Next.js site to a public URL with Vercel.
          </p>
        </a>
      </div>
    </main>
    <footer
      class="jsx-2250570153"
    >
      <a
        class="jsx-2250570153"
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by 
        <img
          alt="Vercel Logo"
          class="jsx-2250570153 logo"
          src="/vercel.svg"
        />
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
